strangelove-ventures/wadesta
# Feat/vite example

## Description

This PR moves the stand-alone vite example into the main graz mono-repo

## Checklist


## Changes

- [x] Added vite-example project to example dir
- [x] Updated pnpm workspaces in root of mono repo

## Testing

- Check out feature branch
- Run `pnpm i`
- Run `pnpm example:vite build`
- Ensure that vite example builds successfully
